packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -299,19 +299,28 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
     }
   };
 
-  private _onGotoToday = (): void => {
+  private _onGotoToday = (selectDateOnClick: boolean): void => {
```

Comment:
Hmm, this is weird up to a click handler. The first parameter of a click handler is an event args. So... wouldn't selectDateOnClick always be truthy?